website/docs/r/database_secret_backend_static_role.md

```diff
@@ -38,6 +38,16 @@ resource "vault_database_secret_backend_static_role" "static_role" {
   rotation_period     = "3600"
   rotation_statements = ["ALTER USER \"{{name}}\" WITH PASSWORD '{{password}}';"]
 }
+
+resource "vault_database_secret_backend_static_role" "static_role" {
```

Comment:
nit:

```suggestion
# configure a static role with schedule-based rotations
resource "vault_database_secret_backend_static_role" "static_role" {
```

And I would suggest a similar comment for the resource block above for period-based rotations